src/frontend/src/pages/FinancePage/ReimbursementRequestForm/ReimbursementFormView.tsx

```diff
@@ -187,19 +193,28 @@ const ReimbursementRequestFormView: React.FC<ReimbursementRequestFormViewProps>
                 <Controller
                   name="expenseTypeId"
                   control={control}
-                  render={({ field: { onChange, value } }) => (
-                    <Select
-                      onChange={(newValue) => onChange(newValue.target.value)}
-                      value={value}
-                      error={!!errors.expenseTypeId}
-                    >
-                      {allExpenseTypes.map((expenseType) => (
-                        <MenuItem key={expenseType.expenseTypeId} value={expenseType.expenseTypeId}>
-                          {expenseType.name}
-                        </MenuItem>
-                      ))}
-                    </Select>
-                  )}
+                  render={({ field: { onChange, value } }) => {
+                    return (
+                      <NERAutocomplete
+                        options={allExpenseTypes.map(expenseTypesToAutocomplete)}
+                        value={
+                          allExpenseTypes
+                            .map(expenseTypesToAutocomplete)
+                            .find((expenseType) => expenseType.label === value) || null
+                        }
+                        placeholder="Expense Type"
+                        onChange={(_event, newValue) => {
+                          if (newValue) {
+                            onChange(newValue.label);
+                          } else {
+                            onChange('');
+                          }
+                        }}
+                        id={'expenseType'}
+                        size={undefined}
```

Comment:
set this to small/medium and see how it looks, but I wouldn't set it as undefined like this